libs/components/lists/src/lib/modules/filter/filter-button.component.scss

```diff
@@ -20,16 +20,64 @@
   .sky-filter-btn-active,
   .sky-filter-btn-active:hover,
   .sky-filter-btn-active:focus {
-    color: initial;
+    background-color: var(--sky-color-background-selected-soft);
     border: none;
-
-    @include modernMixins.sky-theme-modern-button-variant(
-      var(--sky-text-color-action-primary),
-      var(--sky-background-color-input-selected),
-      var(--sky-background-color-primary-dark),
-      var(--sky-background-color-primary-dark),
-      $sky-theme-modern-padding-button-standard
+    box-shadow: inset 0 0 0 var(--sky-border-width-action-base)
```

Comment:
This doesn't appear to match the token in the spec of `--sky-border-width-action-selected-s`